Documentation/ART_NOTES.md

```diff
@@ -0,0 +1,119 @@
+
+# **Art Notes**
+
+Hi! Welcome to the art notes, a page that covers some of the art in Boss Room.
+
+## **Baking Overview**
+
+Boss Room is set up to support baked lighting. Please feel free to use the existing setup or modify it as needed. <br /><br />
+
+### **TL:DR**
+
+* To bake: Go to lighting settings (Window > Rendering > Lighting). Use the FastBaking baking profile & hit Generate Lighting button to bake lights
+* Lower values in baking settings = faster bake (see **⚠ Important ⚠** quote below for more info)
+* Only lights set to Baked mode will contribute to baked lighting
+* Meshes need to have Contribute Global Illumination checked on to receive baked lighting <br /> <br />
+
+### **Baked Lighting & Navigating Settings:**
+* The light explorer window can be found at Window > Rendering > Light Explorer. This window can be used to see all of the lights in your scene(s), plus all of their settings <br />
+
+    ![LightExplorer](Images/LightExplorer.png) <br /><br />
+
+* Each light is set to one of three modes: 
+  
+  * Realtime - not baked, will contribute to realtime lighting, will not contribute to lightmaps
+  * Mxed - baked and realtime, will contribute to realtime lighting, will contribute to lightmaps
+  * Baked - not realtime, will not contribute to realtime, will only contribute to lightmaps
+  
+    ![LightMode](Images/LightMode.png) <br /><br />
+
+### **Lighting Settings**
+
+Lighting settings can be found at Window > Rendering > Lighting
+
+![LightingSettings](Images/LightingPanel.png)
+
+* There are three tabs at the top for navgating your lighting options:
+  * Scene: light baking options for the current scene(s)
+  * Environment: settings for overall environment changes (ambient color, skybox, etc.) in your current scene(s)
+  * Baked Lightmaps: where you can look at your baked light maps if you have baked lighting for your current scene(s) <br /><br />
+  
+> **⚠ Important ⚠**
+  >* The only baking settings that affect **performance of your game** are:
+    >* Lightmap resolution
+    >* Lightmap padding
+    >* Max Lightmap Size
+    >* The Compress Lightmaps checkbox
+    >* The Directional mode 
+    >
+        >![ImportantLightingSettings](Images/ImportantLightingSettings.png)
+  >* All other settings will only affect the **quality of the bake** 
+  >* Higher values in your settings will make baking take longer
+    >* It's recommended to start with very low values for your settings to make baking fast and lighting iteration easy. Then, when you have something you like, you can bump up these settings for a slower, but better quality bake 
+    >* For example, start with a low Lightmap Resolution, like 1, for iteration. Then, bump it up to 6 or 8 for a higher quality bake 
+
+ <br />       
+    
+* You can hover over each setting and a tooltip will descirbe what it does <br /><br />
```

Comment:
```suggestion
* You can hover over each setting and a tooltip will describe what it does <br /><br />
```

Reply:
I realize this was a requested change on the other md file. I made all of my requested changes for this version, since this was the path by Sam.

Just a reminder to remove the other version.